Client/Ecosia/UI/NTP/Cells/NTPBookmarkNudgeCell.swift

```diff
@@ -0,0 +1,182 @@
+        super.init(coder: coder)
+        setup()
+    }
+
+    private func setup() {
+        contentView.addSubview(backgroundCard)
+        backgroundCard.addSubview(badge)
+        backgroundCard.addSubview(closeButton)
+        backgroundCard.addSubview(descriptionLabel)
+        backgroundCard.addSubview(openBookmarksButton)
+        backgroundCard.addSubview(icon)
+        
+        NSLayoutConstraint.activate([
+            backgroundCard.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            backgroundCard.topAnchor.constraint(equalTo: contentView.topAnchor, constant: UX.insetMargin),
+            backgroundCard.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+
+            badge.heightAnchor.constraint(equalToConstant: UX.badgeHeight).priority(.required),
+            badge.topAnchor.constraint(equalTo: backgroundCard.topAnchor, constant: UX.insetMargin),
+            badge.leadingAnchor.constraint(equalTo: backgroundCard.leadingAnchor, constant: UX.insetMargin),
+            
+            closeButton.widthAnchor.constraint(equalToConstant: 44),
+            closeButton.heightAnchor.constraint(equalToConstant: 44),
+            closeButton.trailingAnchor.constraint(equalTo: backgroundCard.trailingAnchor),
+            closeButton.topAnchor.constraint(equalTo: backgroundCard.topAnchor),
+            
+            descriptionLabel.topAnchor.constraint(equalTo: badge.bottomAnchor, constant: UX.insetMargin / 2),
+            descriptionLabel.leadingAnchor.constraint(equalTo: backgroundCard.leadingAnchor, constant: UX.insetMargin),
+            descriptionLabel.trailingAnchor.constraint(equalTo: icon.leadingAnchor, constant: -UX.insetMargin),
+            
+            openBookmarksButton.heightAnchor.constraint(equalToConstant: 32).priority(.required),
+            openBookmarksButton.topAnchor.constraint(equalTo: descriptionLabel.bottomAnchor, constant: UX.insetMargin),
+            openBookmarksButton.leadingAnchor.constraint(equalTo: backgroundCard.leadingAnchor, constant: UX.insetMargin),
+            openBookmarksButton.bottomAnchor.constraint(equalTo: backgroundCard.bottomAnchor, constant: -UX.insetMargin),
+            
+            icon.trailingAnchor.constraint(equalTo: backgroundCard.trailingAnchor, constant: -UX.insetMargin),
+            icon.bottomAnchor.constraint(equalTo: openBookmarksButton.bottomAnchor, constant: 0),
+            icon.widthAnchor.constraint(equalToConstant: 64).priority(.required),
+            icon.heightAnchor.constraint(equalToConstant: 64).priority(.required),
+            
+            backgroundCard.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -UX.insetMargin)
+        ])
+        
+        closeButton.addTarget(self, action: #selector(handleClose), for: .touchUpInside)
+        openBookmarksButton.addTarget(self, action: #selector(handleOpenBookmarks), for: .touchUpInside)
+    }
+    
+    func applyTheme() {}
```

Comment:
Is there a reason for having this class conforming to `NotificationThemeable` if the `applyTheme` isn't utilized?

Reply:
I've missed the thumbing here, also I just learned about the notification which we subscribe to to update the themes, added this. thx!